corehq/apps/app_manager/fixtures/mobile_ucr.py

```diff
@@ -100,7 +100,7 @@ def _report_config_to_fixture(self, report_config, user):
             filter_elem = self._element('filter', attrib={'field': filter_slug})
             option_values = filter_options_by_field[ui_filter.field]
             choices = ui_filter.choice_provider.get_choices_for_values(option_values)
-            choices.sort(key=lambda (value, display): display)
+            choices = sorted(choices, key=lambda choice: choice.display)
```

Comment:
This is just because I'm changing `get_choices_for_values` to return a set. (I figure if we're relying on the caller to sort, returning a list is misleading.)